### PR TITLE
Changelog for 0.1.0

### LINKED ISSUE #38: Update the changelog
### Proposed change

Update the project's changelog to accurately reflect the changes made within the project.

### LINKED ISSUE #45: Initial release of `pytest-jupyterhub`
In preparation for the alpha release of [`pytest-jupyterhub`](https://github.com/jupyterhub/pytest-jupyterhub), this issue will help us keep track of things we need to do to prepare for the release. Feel free to let me know if I have missed any steps.

- [x] #37
- [ ] #38
- [x] #46
- [x] include GitHub workflow badges
- [x] #47

### PR DESCRIPTION
**Pull request by Sheila-nk**
Working towards Initial Release #45 

Closes #38
Closes #45

### CODE REVIEW

File: docs/source/changelog.md [hunk 1]
```diff
@@ -1,19 +1,49 @@
 # Changelog
 
 For detailed changes from the prior release, click on the version number, and
-its link will bring up a GitHub listing of changes. Use `git log` on the
-command line for details.
+its link will bring up a GitHub listing of changes. Use `git log` on the command line for details.
 
-([full changelog](https://github.com/jupyterhub/pytest-jupyterhub/compare/0a6ed6d634f8bd9ae294a367fad1757f521f18b1...3407b07596b0be6cc141a8a671fa8be2257d1c80))
+## [0.1.0](https://github.com/jupyterhub/pytest-jupyterhub/compare/60e995b618c5aec4a950523dc880a65a031769ab...5f535ddf56b3c0953bfa48c9d8d9ff45734f9417) 2023-04-
```
Comment (consideRatio) [inline docs/source/changelog.md:6]: 0a6ed6d634f8bd9ae294a367fad1757f521f18b1 is the initial commit, and 0.1.0 will be the reference to compare with when its released, allowing this to not need further updates later.

```suggestion
## [0.1.0](https://github.com/jupyterhub/pytest-jupyterhub/compare/0a6ed6d634f8bd9ae294a367fad1757f521f18b1...0.1.0) 2023-04-
```

File: docs/source/changelog.md [hunk 2]
```diff
@@ -1,19 +1,51 @@
 # Changelog
 
 For detailed changes from the prior release, click on the version number, and
-its link will bring up a GitHub listing of changes. Use `git log` on the
-command line for details.
+its link will bring up a GitHub listing of changes. Use `git log` on the command line for details.
 
-([full changelog](https://github.com/jupyterhub/pytest-jupyterhub/compare/0a6ed6d634f8bd9ae294a367fad1757f521f18b1...3407b07596b0be6cc141a8a671fa8be2257d1c80))
+## [0.1.0](https://github.com/jupyterhub/pytest-jupyterhub/compare/0a6ed6d634f8bd9ae294a367fad1757f521f18b1...0.1.0) 2023-04-XX
```
Comment (consideRatio) [inline docs/source/changelog.md:6]:
```suggestion
## [0.1.0](https://github.com/jupyterhub/pytest-jupyterhub/compare/0a6ed6d634f8bd9ae294a367fad1757f521f18b1...0.1.0) 2023-04-25
```

This looks ready for a release right now :)

Comment (consideRatio) [reply docs/source/changelog.md:6]: Btw you can add "Closes #38" to the PR description as well

Comment (consideRatio) [reply docs/source/changelog.md:6]: And "Closes #45"!

**Review by consideRatio — APPROVED** [review]
Wieee! Nice work and congratulations on the 0.1.0 release!!

For a changelog PR, an approval in the jupyterhub github org typically means that its okay to self-merge and go for a release (by following RELEASE.md notes).

**Comment by Sheila-nk** [comment]
Yeyyy!! 🚀 🚀  
Thanks for the review! @consideRatio 
Will proceed to merge this PR and then make the release! 🎉

**Pull request merged**